Roguelike/Actors/InventoryUtils/Items/Attacks/BowAttack.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using Roguelike.Core;
+
+namespace Roguelike.Actors.InventoryUtils.Items.Attacks;
+
+public class BowAttack<TAmmo> : IRangeAttack where TAmmo : Actor, ICloneable
+{
+    public TAmmo Arrow { get; set; }
+    public List<Vector2Int> range { get; set; } = new();
```

Comment:
```suggestion
    public List<Vector2Int> Range { get; set; } = new();
```
Поставили бы StyleCop